src/NetscriptWorker.js

```diff
@@ -164,8 +164,17 @@ function runScriptsLoop() {
             var ip = workerScripts[i].serverIp;
             var name = workerScripts[i].name;
 
-            //Free RAM
-            AllServers[ip].ramUsed -= workerScripts[i].ramUsage;
+            //recalculate ram used
+            AllServers[ip].ramUsed = 0;
```

Comment:
Shouldn't this line be before the outer loop, since we're rebuilding the ramUsed on the server?